### PR TITLE
Feature/e UI 3268 use ccd conditional show form directive in read only contexts

### PR DESCRIPTION
**Pull request by kuda1992**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-3268

### Change description ###

1. Remove the `ccdConditionalShow` directive 
2. Refactor the  tests using `ccdConditionalShow` directive

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/shared/components/case-editor/case-edit-submit/case-edit-submit.component.spec.ts [hunk 1]
```diff
@@ -578,6 +585,9 @@ describe('CaseEditSubmitComponent', () => {
         declarations: [
           CaseEditSubmitComponent,
           IsCompoundPipe,
+          CcdPageFieldsPipe,
+          FieldsFilterPipe,
+          ReadFieldsFilterPipe,
```
Comment (connorpgpmcelroy) [inline src/shared/components/case-editor/case-edit-submit/case-edit-submit.component.spec.ts:590]: Why are they listed differently in these two sections? - Ccd... at bottom of first section, top of second

File: src/shared/components/case-history/case-history.component.spec.ts [hunk 2]
```diff
@@ -120,12 +126,12 @@ describe('CaseHistoryComponent', () => {
...
-          { provide: OrderService, useValue: orderService },
-          { provide: CaseNotifier, useValue: caseNotifier },
-          { provide: CaseHistoryService, useValue: caseHistoryService },
-          { provide: Router, useValue: router }
+          {provide: AlertService, useValue: alertService},
+          {provide: ActivatedRoute, useValue: mockRoute},
+          {provide: OrderService, useValue: orderService},
+          {provide: CaseNotifier, useValue: caseNotifier},
+          {provide: CaseHistoryService, useValue: caseHistoryService},
+          {provide: Router, useValue: router}
```
Comment (connorpgpmcelroy) [inline src/shared/components/case-history/case-history.component.spec.ts:134]: Not sure about formatting here - think spaces either side are better and the current consistent formatting but might have changed due to prettier?

File: src/shared/components/case-viewer/case-viewer.module.ts [hunk 3]
```diff
@@ -28,23 +28,24 @@ import { CaseViewComponent } from './case-view/case-view.component';
...
+        PaletteModule,
+        LabelSubstitutorModule,
+        CaseEditorModule,
+        PipesModule,
+        ConditionalShowModule,
+        CaseHistoryModule,
+        EventLogModule,
+        MatTabsModule,
+        ComplexModule
+    ],
```
Comment (connorpgpmcelroy) [inline src/shared/components/case-viewer/case-viewer.module.ts:48]: Spacing issue here?

File: src/shared/components/palette/complex/ccd-collection-table-value-case-fields.pipe.spec.ts [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import { CcdCollectionTableCaseFieldsFilterPipe } from './ccd-collection-table-value-case-fields.pipe';
+
+describe('CddFieldsKeyValueCaseFieldPluckPipe', () => {
```
Comment (connorpgpmcelroy) [inline src/shared/components/palette/complex/ccd-collection-table-value-case-fields.pipe.spec.ts:3]: Why does this say PluckPipe?

File: src/shared/components/workbasket-filters/workbasket-filters.component.html [hunk 5]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (connorpgpmcelroy) [inline src/shared/components/workbasket-filters/workbasket-filters.component.html:1]: Is this change necessary?

**Review by connorpgpmcelroy — CHANGES_REQUESTED** [review]
Have some questions/comments - mostly about formatting

File: src/shared/components/palette/complex/ccd-collection-table-value-case-fields.pipe.spec.ts (hunk 4 above)
Comment (kuda1992) [reply src/shared/components/palette/complex/ccd-collection-table-value-case-fields.pipe.spec.ts:3]: Good spot I renamed the pipe guess It didn't change the string value

File: src/shared/components/workbasket-filters/workbasket-filters.component.html (hunk 5 above)
Comment (kuda1992) [reply src/shared/components/workbasket-filters/workbasket-filters.component.html:1]: I must have been logging something and create an extra line. Removed
